src/content_script.css

```diff
@@ -1,335 +1,336 @@
 /* Button Replacement */
 .fbc-loginButton {
-	background-size: contain;
-	background-position: center;
-	background-repeat: none;
-	cursor: pointer;
+  background-size: contain;
```

Comment:
This entire file (src/content_script.css) should be reset/removed from this PR. All the line charges are just to white space. Can you please reset this?